test/ui-test/src/screens/MainUi.py

```diff
@@ -0,0 +1,23 @@
+# ******************************************************************************
+# Status.im
+# *****************************************************************************/
+# /**
+# * \file    MainUi.py
+# *
+# * \date    July 2022
+# * \brief   General UI functions that are either global or not part of a specific module
+# *****************************************************************************/
+
+
+from enum import Enum
+import time
+from drivers.SquishDriver import *
+
+class MainUi(Enum):
+    MODULE_WARNING_BANNER = "moduleWarning_Banner"
+
+# Wait for the banner to disappear otherwise the click might land badly
+def wait_for_banner_to_disappear():
+    [bannerLoaded, _] = is_loaded_visible_and_enabled(MainUi.MODULE_WARNING_BANNER.value)
+    if (bannerLoaded):
+        time.sleep(5)
```

Comment:
Just wondering if we could verify here if the banner is really invisible?

Reply:
I didn't mean anything sophisticated here. What I wanted to suggest is:
```
    if (bannerLoaded):
        time.sleep(5)
        verify_invisible(MainUi.MODULE_WARNING_BANNER.value)
```
So that tests can fail with an appropriate message, rather than one step further.

Reply:
> I didn't mean anything sophisticated here. What I wanted to suggest is:

I want to remove the sleep and use wait for an object instead and save some time while making it reliable. Now that I did the first step of moving it to a function the next will be to improve it